tests/auth_plugin_test.php

```diff
@@ -843,6 +844,84 @@ public function test_ipmismatch_exception_thrown_if_ip_is_incorrect() {
     }
 
     /**
+     * Test that IP address mismatch exception gets thrown if incorrect IP.
```

Comment:
Is it correct test description?

Reply:
No, was copy and pasted and not changed, fixed.